sources/detail/SplaLibraryPrivate.hpp

```diff
@@ -28,37 +28,41 @@
 #ifndef SPLA_SPLALIBRARYPRIVATE_HPP
 #define SPLA_SPLALIBRARYPRIVATE_HPP
 
+#include <boost/compute/device.hpp>
+#include <boost/compute/system.hpp>
 #include <spla-cpp/SplaLibrary.hpp>
 #include <spla-cpp/SplaDescriptor.hpp>
 #include <taskflow/taskflow.hpp>
 #include <expression/SplaExpressionManager.hpp>
```

Comment:
It would be great, if you reorder includes here. 
Consider `spla-cpp` files first, `spla files from sources`, then `dependencies` and then `std` includes. 
In the project we are going to follow this rule.

```cpp
#include <spla-cpp/SplaLibrary.hpp>
#include <spla-cpp/SplaDescriptor.hpp>
#include <expression/SplaExpressionManager.hpp>`
#include <boost/compute/device.hpp>
#include <boost/compute/system.hpp>
#include <taskflow/taskflow.hpp>
```